io.github.nroduit.Weasis.yaml

```
@@ -0,0 +1,56 @@
+app-id: io.github.nroduit.Weasis
+runtime: org.freedesktop.Sdk
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+base-version: '21.08'
+sdk-extensions: 
+  - org.freedesktop.Sdk.Extension.openjdk11
+command: viewer-linux.sh
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --share=network
+  - --filesystem=home
+  - --env=PATH=/app/jre/bin:/app/bin:/usr/bin
+  - --env=JAVA_HOME=/app/jre
+modules:
+
+  - name: openjdk
+    buildsystem: simple
+    build-commands:
+      - /usr/lib/sdk/openjdk11/install.sh
+
+  - name: weasis
+    buildsystem: simple
+    cleanup:
+      - /weasis-*
+      - .m2
+    build-options:
+      #build-args:
+      #  - --share=network
+      env:
+        PATH: /app/bin:/usr/bin:/usr/lib/sdk/openjdk11/bin
+        MAVEN_OPTS: -Dmaven.repo.local=.m2/repository
+        JAVA_HOME: /usr/lib/sdk/openjdk11/jvm/openjdk-11
+    build-commands:
+      - mvn clean install
+      - cd weasis-distributions && mvn clean package -Dportable=false -Dmaven.repo.local=../.m2/repository
+      - unzip weasis-distributions/target/portable-dist/weasis-portable.zip
+      - mv weasis /app
+      - sed -i 's/curPath=.*/curPath=\/app\/weasis/' viewer-linux.sh
+      - install -Dm755 -t /app/bin viewer-linux.sh
+      - install -D -t /app/share/icons/hicolor/64x64/apps/io.github.nroduit.Weasis.png weasis-distributions/script/resources/linux/Weasis.png
```

128x128 is required so this might actually cause a failure

Otherwise your install command is wrong: `-t` argument is a directory.

```suggestion
      - install -D weasis-distributions/script/resources/linux/Weasis.png /app/share/icons/hicolor/64x64/apps/io.github.nroduit.Weasis.png
```